docker-compose.yml

```diff
@@ -624,8 +625,6 @@ services:
       - _APP_DB_PASS
       - _APP_LOGGING_PROVIDER
       - _APP_LOGGING_CONFIG
-      - _APP_MIGRATIONS_FIREBASE_CLIENT_ID
-      - _APP_MIGRATIONS_FIREBASE_CLIENT_SECRET
```

Comment:
why was this removed?

Reply:
Since we're never rolling out firebase OAuth so I just removed it while I was there